commands/daily.py

```diff
@@ -30,6 +30,13 @@ async def daily(self, ctx):
             multiplier = 1
 
         if last_advent < current_time:
+            
+            def update_advent(user_id: str):
+                next_advent =  current_time + ((60 * 60) * 20)
+                sql.execute_query("UPDATE Members SET dailyRewardClaimed = %s WHERE UserID = %s " % (str(next_advent), str(user_id)))

```

Comment:
Why are we converting something to a string that's already set as a string above? Again to my previous comment, just use ctx.author.id instead of passing in a value, no need to store something that is already made available